docs/submitting.md

```diff
@@ -1,35 +1,41 @@
 Submitting a paper to JOSS
 ==========================
 
-If you've already licensed your code and have good documentation then we expect that it should take less than an hour to prepare and submit your paper to JOSS.
+If you've already developed a fully featured research code, released it under an OSI-approved license, and written good documentation and tests, then we expect that it should take perhaps an hour or two to prepare and submit your paper to JOSS.
+But please read these instructions carefully for a streamlined submission.
 
 ## Typical paper submission flow
 
-Before you submit we need you to:
+Before you submit, you should:
 
-- Make software available in an open repository (GitHub, Bitbucket etc.) and include an [OSI approved open source license](https://opensource.org/licenses).
-- Make sure that the software complies with the [JOSS review criteria](review_criteria.html).
-- Author a short Markdown paper `paper.md` with a title, summary, author names, affiliations, and key references. See an example [here](https://github.com/arfon/fidgit/tree/master/paper).
-- (Optional) create a metadata file and include it in your repository describing your software. [This script](https://gist.github.com/arfon/478b2ed49e11f984d6fb) automates the generation of this metadata.
+- Make your software available in an open repository (GitHub, Bitbucket, etc.) and include an [OSI approved open source license](https://opensource.org/licenses).
+- Make sure that the software complies with the [JOSS review criteria](review_criteria.html). In particular, your software should be full-featured, well-documented, and contain tests for checking correctness.
+- Write a short paper in Markdown format using `paper.md` as file name, including a title, summary, author names, affiliations, and key references. See our [example paper](https://github.com/arfon/fidgit/tree/master/paper) to follow the correct format.
```

Comment:
```suggestion
- Write a short paper in Markdown format using `paper.md` as file name, including a title, summary, author names, affiliations, and key references. See our [example paper](#example-paper-and-bibliography) to follow the correct format.
```

Reply:
This updates the example paper link to point to https://joss.readthedocs.io/en/latest/submitting.html#example-paper-and-bibliography which is a more fully-featured example paper.